### ISSUE TITLE
EN 15804+A2, chapter 7.1, a), the name and address of the manufacturer(s)

### ISSUE THREAD

**Issue by RobertSpang** [InDataWG/ILCD-EPD-Data-Format#20]
Implementation status:
- [x] schema
- [x] examples
- [x] reviewed & approved internally
- [ ] documentation
- [ ] validation rules

### I raise this change request because...

-- [ ] the ILCD+EPD format lacks compliance with regualtions.
-- [X] the ILCD+EPD format lacks conformaty with standards.
-- [ ] the ILCD+EPD format needs to be improved regarding existing format or master data elements.
-- [ ] there is another problem or use case I´m trying to address.

### Problem description

the name and address of the manufacturer(s) shall be mentioned.
Is most probably not covered at the moment, since it could vary from C2.3 data set generator and  C3.8 owner of the data set

### Solution description

Check if new field is needed and how we would deal with large assiciations

### Relevance or added value

Medium relevance

**Comment by RobertSpang** [comment]
We understand, that the format already contains a field which is "owner of the EPD" and in the field "use advice for the dataset" the sites covered shall be described. So from our perspective no action is required at the moment.

**Comment by okworx** [comment]
The discussion arrives at a point where both the names and addresses of the manufacturers (legal entity) are to be declared as well as the production sites (geo coordinates).

Proposal: we are allowing a list of manufacturers and nested for each manufacturer the productions sites can be declared using geo coordinates and optionally street address. Site ID and manufacturer ID to be included. 


addresses https://github.com/InDataWG/ILCD-EPD-Data-Format/issues/18 and https://github.com/InDataWG/ILCD-EPD-Data-Format/issues/23

**Comment by 1aun** [comment]
### Use for *(incl. reference to standard, section/subsection)*

* EN 15804+A2, chapter 7.1, a), the name and address of the manufacturer(s)

### Don't user for

* Dataset generator
* Owner of the dataset

### Suggested Name *(if e.g. a single new field is requested)*

manufacturers

### Suggested Locations within ILCD

* DatasetInformation

### Description of context

Goods and products have a manufacturer, sometimes even more than one.
That is the case, if a product is developed

* simultaneously in more than one plant
* in parts in different places.

### Occurrence

*Mandatory or optional*


### Cardinality

*One or many*


### Content / Data

***Description of what the data should contain** (describing the data is better than describing an implementation). If the data is complex and consists of multiple aspects, describe them separately. Think about resp. occurrences and cardinalities and describe data types/constraints whenever these infos are already clear (e.g. "must be a positive integer")*.


### Constraints

* *List the constraints that need to be enforced (validation errors)*

**Comment by okworx** [comment]
Regexes for checking OLC: https://github.com/google/open-location-code/blob/main/Documentation/Reference/App_Developers.md

**Comment by okworx** [comment]
suggested markup:

```xml
<epd24:manufacturers>
  <epd24:manufacturer>
     <epd24:contact type="contact data set" refObjectId="12345678-1234-1234-1234-123456789012" version="00.00.000">
        <common:shortDescription xml:lang="en">ACME</common:shortDescription>
     </epd24:contact>
     <epd24:sites>
        <epd24:site>
           <epd24:name>ACME Base camp</epd24:name>
           <epd24:facilityIdentifier>KELLNERDOM</epd24:facilityIdentifier>
           <epd24:geoCode>DE</epd24:geoCode>
           <epd24:olc>9F28WXR4+FW2</epd24:olc>
        </epd24:site>
     </epd24:sites>
  </epd24:manufacturer>
</epd24:manufacturers>
```

**Comment by okworx** [comment]
@ domain experts: please review and approve (by giving a thumbs up 👍) or else comment

**Comment by szwerenz** [comment]
Just to check that multiple entries are possible, for multisite EPDs and Association EPDs.
Otherwise the proposal is good.

**Comment by okworx** [comment]
> Just to check that multiple entries are possible, for multisite EPDs and Association EPDs. Otherwise the proposal is good.

Yes, of course, multiple `manufacturer` elements can be contained in the `manufacturers` element.
